lib/generators/doorkeeper/templates/migration.rb

```diff
@@ -27,7 +27,7 @@ def change
     create_table :oauth_access_tokens do |t|
       t.integer  :resource_owner_id
       t.integer  :application_id
-      t.string   :token,             null: false
+      t.text     :token,             null: false
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.